### PR TITLE
Regeneration of services for new discovery endpoints

### PR DESCRIPTION
**Pull request by anweshan**

### CODE REVIEW

File: packages/assistant-v1/actions/create-dialog-node.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ const extend = require('extend');
  * carriage return, newline, or tab characters, and it must be no longer than 2048 characters.
  * @param {string} [params.parent] - The ID of the parent dialog node.
  * @param {string} [params.previous_sibling] - The ID of the previous dialog node.
- * @param {Object} [params.output] - The output of the dialog node. For more information about how to specify
+ * @param {DialogNodeOutput} [params.output] - The output of the dialog node. For more information about how to specify
```
Comment (mediumTaj) [inline packages/assistant-v1/actions/create-dialog-node.js:47]: Should these stay generic objects? I think `DialogNodeOutput` has `additionalProperties`.

File: packages/assistant-v1/actions/update-dialog-node.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ const extend = require('extend');
  * carriage return, newline, or tab characters, and it must be no longer than 2048 characters.
  * @param {string} [params.new_parent] - The ID of the parent dialog node.
  * @param {string} [params.new_previous_sibling] - The ID of the previous sibling dialog node.
- * @param {Object} [params.new_output] - The output of the dialog node. For more information about how to
+ * @param {DialogNodeOutput} [params.new_output] - The output of the dialog node. For more information about how to
```
Comment (mediumTaj) [inline packages/assistant-v1/actions/update-dialog-node.js:49]: Should these stay generic objects? I think `DialogNodeOutput` has `additionalProperties`.

File: packages/conversation-v1/actions/create-dialog-node.js [hunk 3]
```diff
@@ -44,7 +44,7 @@ const extend = require('extend');
  * carriage return, newline, or tab characters, and it must be no longer than 2048 characters.
  * @param {string} [params.parent] - The ID of the parent dialog node.
  * @param {string} [params.previous_sibling] - The ID of the previous dialog node.
- * @param {Object} [params.output] - The output of the dialog node. For more information about how to specify
+ * @param {DialogNodeOutput} [params.output] - The output of the dialog node. For more information about how to specify
```
Comment (mediumTaj) [inline packages/conversation-v1/actions/create-dialog-node.js:47]: Should these stay generic objects? I think `DialogNodeOutput` has `additionalProperties`.

File: packages/conversation-v1/actions/update-dialog-node.js [hunk 4]
```diff
@@ -46,7 +46,7 @@ const extend = require('extend');
  * carriage return, newline, or tab characters, and it must be no longer than 2048 characters.
  * @param {string} [params.new_parent] - The ID of the parent dialog node.
  * @param {string} [params.new_previous_sibling] - The ID of the previous sibling dialog node.
- * @param {Object} [params.new_output] - The output of the dialog node. For more information about how to
+ * @param {DialogNodeOutput} [params.new_output] - The output of the dialog node. For more information about how to
```
Comment (mediumTaj) [inline packages/conversation-v1/actions/update-dialog-node.js:49]: Should these stay generic objects? I think `DialogNodeOutput` has `additionalProperties`.

File: packages/visual-recognition-v3/actions/update-classifier.js [hunk 5]
```diff
@@ -70,7 +70,7 @@ function main(params) {
       _params.headers,
       { 'User-Agent': 'openwhisk' }
     );
-    const fileParams = ['classname_positive_examples', 'negative_examples'];
+    const fileParams = ['negative_examples'];
```
Comment (mediumTaj) [inline packages/visual-recognition-v3/actions/update-classifier.js]: Do we not need positive examples here?

**Review by mediumTaj — CHANGES_REQUESTED** [review]
A couple of questions and can you update your readme with

```
### Getting credentials
To find out which authentication to use, view the service credentials. You find the service credentials for authentication the same way for all Watson services:

1. Go to the IBM Cloud [Dashboard](https://console.bluemix.net/dashboard/apps?category=ai) page.
1. Either click an existing Watson service instance or click [**Create resource > AI**](https://console.bluemix.net/catalog/?category=ai) and create a service instance.
1. Copy the `url` and either `apikey` or `username` and `password`. Click **Show** if the credentials are masked.
```

File: packages/assistant-v1/actions/update-dialog-node.js (hunk 2 above)
Comment (anweshan) [reply packages/assistant-v1/actions/update-dialog-node.js:49]: It is a generic object, that is the documentation

File: packages/visual-recognition-v3/actions/update-classifier.js (hunk 5 above)
Comment (anweshan) [reply packages/visual-recognition-v3/actions/update-classifier.js]: good catch

File: README.md [hunk 6]
```diff
@@ -81,6 +86,13 @@ You supply either an IAM service **API key** or an **access token** with the par
 
 > _NOTE_: Authenticating with the `X-Watson-Authorization-Token` header or the `watson-token` query param is now deprecated. The token continues to work with Cloud Foundry services, but is not supported for services that use Identity and Access Management (IAM) authentication. For details see [Authenticating with IAM tokens](https://console.bluemix.net/docs/services/watson/getting-started-iam.html#iam)
 
+### Getting credentials
+To find out which authentication to use for passing in authenticatin parameters, view the service credentials. You find the service credentials for authentication the same way for all Watson services:
+
+1. Go to the IBM Cloud [Dashboard](https://console.bluemix.net/dashboard/apps?category=ai) page.
+1. Either click an existing Watson service instance or click [**Create resource > AI**](https://console.bluemix.net/catalog/?category=ai) and create a service instance.
+1. Copy the `url` and either `apikey` or `username` and `password`. Click **Show** if the credentials are masked.
+
```
Comment (mediumTaj) [inline README.md]: Do we need this second instance of `Getting credentials`?

**Review by mediumTaj — CHANGES_REQUESTED** [review]

**Comment by mediumTaj** [comment]
Tests are failing right now too

**Review by mediumTaj — APPROVED** [review]

**Pull request merged**